e2e/tests/imports/testdata/conditional/devspace.yaml

```diff
@@ -3,13 +3,18 @@ name: base
 
 imports:
   - path: ./${IMPORT1_PATH}
+  - path: ./import.yaml
 
 vars:
   IMPORT1_PATH:
     source: env
     default: import1.yaml
+  TODAYS_MESSAGE:
+    source: command
+    command: echo ${NEW_MESSAGE}
```

Comment:
Can we add a case for when both have a var with the same name, where the importer sets the value to `""`?

I suspect that the imported value would be used over the local value, which may be unexpected.